blockstore/pom.xml

```diff
@@ -19,26 +19,16 @@
             <groupId>org.apache.brooklyn</groupId>
             <artifactId>brooklyn-locations-jclouds</artifactId>
             <version>${brooklyn.version}</version>
-            <exclusions>
-                <exclusion>
-                    <groupId>org.apache.jclouds.api</groupId>
-                    <artifactId>openstack-nova</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.apache.jclouds</groupId>
-                    <artifactId>jclouds-core</artifactId>
-                </exclusion>
-            </exclusions>
         </dependency>
         <dependency>
-            <groupId>io.cloudsoft.jclouds</groupId>
+            <groupId>org.apache.jclouds</groupId>
```

Comment:
We can delete this one, and just rely on getting it via `brooklyn-locations-jclouds`